Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,51 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+## Node Loss
```

Comment:
For another PR, it would be good to add a TOC to the top of the doc like we have in ceph-common-issues.md, so this topic would be easier to find.

Reply:
okay I'll do it as a follow up PR